recipe/meta.yaml

```diff
@@ -11,20 +11,18 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true # [py<37 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
-  build:
-    - nodejs >=16
   host:
-    - python >=3.7
+    - python
     - pip
-    - jupyter
```

Comment:
why removed ? 
https://github.com/jupyter-widgets/ipyleaflet/blob/da6701f60d766961b2f7906bee1e1b0fea899369/pyproject.toml#L3

Reply:
There is no `jupyter`. 
`jupyterlab` is not needed to build the package

Reply:
correct